### DISCUSSION TITLE
【Keyball44】キーソケットのランドが剥離してしまいました。

### DISCUSSION THREAD

**Discussion by Laplus8** [Yowkees/keyball#278] [Q&A]
キーソケットのランドが剥離してしまい、キーを押下しても反応しません。解決策を教えて頂けますと幸いです。
宜しくお願い致します。

![image](https://github.com/Yowkees/keyball/assets/139874476/cd24b0ac-a945-4ec0-8bde-143644b45d55)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。状況了解しました。

下記画像の同色どうし（３本）をエナメル線等でショートしていただければ、復活できます。
なお、キーソケットの保持力が弱くなっていますので、両面テープ等で張り付けてから作業されることをお勧めします。

以上、お手数ですがご対応お願いいたします。

![kb44_keyfix](https://github.com/Yowkees/keyball/assets/84209772/155a37e1-5823-41b4-8aee-24fef35e8a9d)

**Comment by Laplus8** [comment]
お世話になっております。
ご返信ありがとうございました。
27番のキースイッチは無事動くようになりました。
16番がタッチ通りに動かず左側のランドも剥離してしまっているかもしれません。
再度ご教授頂けますと幸いです。
重ねてで申し訳ありませんがお時間のある時にご確認の程、お願い致します。

![image](https://github.com/Yowkees/keyball/assets/139874476/ea1e2859-d052-46c1-94ef-c64777734e3d)
